src/main/topLevelPages/myDashboard/latestContributions/LatestContributions.tsx

```diff
@@ -154,6 +154,7 @@ const LatestContributions = ({ limit, spaceMemberships }: LatestContributionsPro
         key={activity.id}
         activity={activity as ActivityLogResultType}
         avatarUrl={activity.triggeredBy.profile.avatar?.uri}
+        avatarName={activity.triggeredBy.profile.displayName}
```

Comment:
I'd call it avatarAlt, or receive an object like you did before `{ url: string, alt: string }`